doc/examples/opencv/Dockerfile

```diff
@@ -1,6 +1,4 @@
-# Our image must derive from pachyderm/job-shim, otherwise Pachyderm won't be
-# able to expose data to it.
-FROM pachyderm/job-shim
+FROM ubuntu:14.04
```

Comment:
None other than that we know that it works with 14.04 and I don't want to bother debugging the opencv example in case it somehow breaks on 16.04.